docs/config-description.md

```diff
@@ -1,452 +1,600 @@
 - **Default value**: None
 - **Constraints**: None
- -   **Description**: Directory path for log files.
-### Key: log_rotation_size
+- **Description**: The directory path for the log files.
+
+### log_rotation_size
+
 - **Required**: True
 - **Type**: int
-- **Default value**: 2048
-- **Constraints**: The minimum value is `1`. The maximum value is `4294967295`
- -   **Description**: Log rotation size in megabytes. When the log file reaches this particular size, a new log file starts.
-### Key: log_directory_max_size
+- **Default value**: `2048`
+- **Constraints**: The minimum value is `1`. The maximum value is `4294967295`.
+- **Description**: The log rotation size in megabytes. When the log file reaches this particular size, a new log file starts.
+
+### log_directory_max_size
+
 - **Required**: True
 - **Type**: int
-- **Default value**: 51200
-- **Constraints**: The minimum value is `1`. The maximum value is `4294967295`
- -   **Description**: Maximum size of the log directory in megabytes.
-### Key: log_rotation_hour_interval
+- **Default value**: `51200`
+- **Constraints**: The minimum value is `1`. The maximum value is `4294967295`.
+- **Description**: The maximum size of the log directory in megabytes.
+
+### log_rotation_hour_interval
+
 - **Required**: True
 - **Type**: int
-- **Default value**: 12
-- **Constraints**: The minimum value is `1`. The maximum value is `4294967295`
- -   **Description**: Interval in hours for log rotation. If the current log file reaches this value in logging, a new log file starts.
-### Key: log_tag_style
+- **Default value**: `12`
+- **Constraints**: The minimum value is `1`. The maximum value is `4294967295`.
+- **Description**: Represents the interval (in hours) for log rotation. If the current log file reaches this value in logging, a new log file starts.
+
+### log_tag_style
+
 - **Required**: True
 - **Type**: string
-- **Default value**: none
-- **Constraints**: The value must be one of the following: `int, uint, null, none, uuid`
- -   **Description**: Style for log tags.
-### Key: extractor_threads
+- **Default value**: `none`
+- **Constraints**: The value must be one of the following: `int`, `uint`, `null`, `none`, `uuid`.
+- **Description**: The style for log tags.
+
+### extractor_threads
+
 - **Required**: True
 - **Type**: int
-- **Default value**: 1
-- **Constraints**: The minimum value is `0`. The maximum value is `4294967295`
- -   **Description**: Number of extractor threads.
-### Key: read_only
+- **Default value**: `1`
+- **Constraints**: The minimum value is `0`. The maximum value is `4294967295`.
+- **Description**: The number of extractor threads.
+
+### read_only
+
 - **Required**: True
 - **Type**: boolean
-- **Default value**: True
+- **Default value**: `True`
 - **Constraints**: None
- -   **Description**: Indicates if the server should have read-only privileges.
-### Key: txn_threshold
+- **Description**: Indicates if the server should have read-only privileges.
```

Comment:
This seems a bit off. The setting indicates whether this instance of Clio is *not* allowed to participate in the ETL process as a writer node (writing ledger data to the DB)